.vscode/arduino.json

```diff
@@ -0,0 +1,5 @@
+{
+    "board": "AZ3166:stm32f4:MXCHIP_AZ3166",
```

Comment:
Is this an ID for a particular board?

Reply:
It seems that the .vscode file has contents about the other files that I edited through VSCode. I uploaded a script to the Azure IoT Board and this is the id of that board. So, do you want me to keep this file, or remove it?

Reply:
Seems like it should be removed.

Reply:
Yeah sure.